src/app/static/src/js/components/runWorkflow/runWorkflowTabs.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+    <div id="workflow-container" class="mt-3">
+        <div class="row">
+            <div class="col-12 col-md-4 col-xl-3">
+                <div class="sidebar pb-4 pb-md-0">
+                    <nav class="pl-0 pr-0 pr-md-4 navbar navbar-light">
+                        <button type="button" class="d-md-none navbar-toggler" data-toggle="collapse"
+                                data-target="#sidebar">
+                            <span class="navbar-toggler-icon"></span>
+                        </button>
+                        <div class="d-md-block mt-4 mt-md-0 collapse navbar-collapse" id="sidebar">
+                            <ul class="nav flex-column list-unstyled mb-0">
+                                <div id="workflow-title">Workflows</div>
+                                <li class="nav-item">
+                                    <a id="run-workflow-link" class="nav-link active" data-toggle="tab" role="tab" href="#"
+                                       @click="switchTab('runWorkflow')">Run</a>
```

Comment:
Following from above:
```suggestion
                                       @click="switchTab('runWorkflow')">Run workflow</a>
```